File: 001.py
# Multiples of 3 or 5
# Find sum of all multiples of 3 or 5 below 1000

def sol(x):
    ans = 0
    for i in range(x):
        if i%5==0 or i%3==0:
            ans+=i
    print(ans)

sol(1000)

# ans = 233168

File: 001.cpp
// Problem 1 - Multiples of 3 or 5
// Find sum of all multiples of 3 or 5 below 1000

#include <iostream>

int sol(int num) {
    int ans = 0;
    for (int i = 0; i < num; ++i) {
        if (i % 5 == 0 || i % 3 == 0) {
            ans += i;
        }
    }

    return ans;
}

int main() {
    int outputNumber = sol(1000);
    std::cout << "Solution: " << outputNumber << std::endl;

    return 0;
}

// ans = 233168